.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+
+env:
+  - TYPO3_BRANCH=TYPO3_6-2
+  - TYPO3_BRANCH=TYPO3_7-0
```

Comment:
However I personnaly think that we should test for retro-compatibility if the extension declare compatibility as a range from 6.2 to 7.X. This is not the case for Media here, so it doesn't really matter.

But, for example, it is the case of the extension Newsletter. It declares [compatibility for the whole range](https://github.com/Ecodev/newsletter/blob/master/ext_emconf.php#L29), so in order to guaratuee (as much as possible) what we declared we test on [all possible branches](https://github.com/Ecodev/newsletter/blob/master/.travis.yml#L7).